charts/fluid/fluid/templates/controller/jindoruntime_controller.yaml

```diff
@@ -11,11 +10,19 @@ spec:
   selector:
     matchLabels:
       control-plane: jindoruntime-controller
+  {{ if .Values.runtime.jindo.enabled -}}
   replicas: {{ .Values.runtime.jindo.replicas }}
+  {{- else }}
+  replicas: 0
+  {{- end }}
   template:
     metadata:
       labels:
         control-plane: jindoruntime-controller
+      annotations:
+      {{ if gt (.Values.runtime.jindo.replicas | int) 1 -}}
+        replicas: {{ .Values.runtime.jindo.replicas | quote }}
```

Comment:
Maybe `expectedReplicas` is a more precise way to describe this field?

Reply:
I think the replicas in spec is already has the meaning `expected`.